Standards/scs-XXXX-vN-taxonomy-of-failsafe-levels.md

```diff
@@ -0,0 +1,264 @@
+
+### Failure Scenarios
+
+The following failure scenarios have been considered for the proposed failsafe levels.
+For each failure scenario, we estimate the probability of occurence and the (worst case) damage caused by the scenario.
+Furthermore, the corresponding minimum failsafe level covering that failure scenario is given.
+
+<!--
+TODO: define the meaning of our probabilities
+-->
+
+#### Hardware Related
+
+| Failure Scenario | Probability | Consequences | Failsafe Level Coverage |
+|----|-----|----|----|
+| Disk Failure | High | Permanent data loss in this disk. Impact depends on type of lost data (data base, user data) | L1 |
+| Host Failure (without disks) | Medium to High | Permanent loss of functionality and connectivity of host (impact depends on type of host) | L1 |
+| Host Failure | Medium to High | Data loss in RAM and temporary loss of functionality and connectivity of host (impact depends on type of host) | L1 |
+| Rack Outage | Medium | Outage of all nodes in rack | L2 |
+| Network router/switch outage | Medium | Temporary loss of service, loss of connectivity, network partitioning | L2 |
+| Loss of network uplink | Medium | Temporary loss of service, loss of connectivity | L3 |
+| Power Outage (Data Center supply) | Medium | Temporary outage of all nodes in all racks | L3 |
+
+#### Software Related
+
+| Failure Scenario | Probability | Consequences | Failsafe Level Coverage |
+|----|-----|----|----|
+| Cyber threat | High | permanent loss or compromise of data on affected Disk and Host | L1 |
+| Software bug (major) | Low | permanent loss or compromise of data that trigger the bug up to data on the whole physical machine | L1 |
+| Software bug (minor) | High | temporary or partial loss or compromise of data | L1 |
+
+<!--
+TODO: how do we arrive at these probabilities?
+-->
+
+#### Environmental
+
+Note that probability for these scenarios is dependent on the location.
+
+| Failure Scenario | Probability | Consequences | Failsafe Level Coverage |
+|----|-----|----|----|
+| Fire | Medium | permanent Disk and Host loss in the affected zone | L3 |
+| Flood | Low | permanent Disk and Host loss in the affected region | L4 |
+| Earthquake | Very Low | permanent Disk and Host loss in the affected region | L4 |
+| Storm/Tornado | Low | permanent Disk and Host loss in the affected region | L4 |
+
+#### Human Interference
+
+| Failure Scenario | Probability | Consequences | Failsafe Level Coverage |
+|----|-----|----|----|
+| Minor operating error | High | Temporary outage | L1 |
+| Major operating error | Low | Permanent loss of data | L3 |
+
+
+## Consequences
+
+Using the definition of levels established in this decision record throughout all SCS standards would allow readers to understand up to which level certain procedures or aspects of resources (e.g. volume types or a backend requiring redundancy) would protect their data and/or resource availability.
+
+### General Terms
```

Comment:
Were these tables the glossary before? Because it looks a little bit like it and would explain anjas comment and my answer. 

I don't get completely what you want to achieve with having these down here? what should this whole part of the decision record about?